core/modules/user/css/user.css

```diff
@@ -68,6 +72,18 @@
   visibility: hidden;
   display: none;
 }
+.password-strength-title {
+  float: left; /* LTR */
+}
+.password-strength-text {
+  padding-left: 5px; /* LTR */
+}
+[dir="rtl"] .password-strength-title {
+  float: right;
+}
+[dir="rtl"] .password-strength-text {
+  padding-right: 5px; /* LTR */
```

Comment:
The padding-left needs to be set back to 0 here.

Reply:
Yep, ...and that ` /* LTR */` should not be there. Also fixed.